src/components/TraderCard/TraderCardBody/TraderCardBody.js

```diff
@@ -74,9 +78,9 @@ const TraderCard = (props) => {
       <div className="traderCardTooltip">
         <div>
           {formatFloat2Dec(tooltipItem.yLabel)}{" "}
-          {isCopyTrading ? "%" : <FormattedMessage id="srv.followers" />}
+          {copyTraders || profitSharingProvider ? "%" : <FormattedMessage id="srv.followers" />}
```

Comment:
Instead of all the `copyTraders || profitSharingProvider` below it could have been interesting to create a bool like `isTrader` for example. But it's fine for now.

Reply:
I left it like this so maybe If we need to change any text for profit sharing

Reply:
Hmm I was only referring to the booleans, it's a bit tedious to repeat `copyTraders || profitSharingProvider` everywhere. One boolean makes more sense to me, so it can be updated easily in the future.